src/main/java/nl/andrewl/emaildatasetbrowser/view/tag/TagEditDialog.java

```diff
@@ -50,31 +53,40 @@ private Container buildUI() {
 
 		JPanel fieldsPanel = new JPanel(new BorderLayout(5, 5));
 		fieldsPanel.add(new LabelledField("Name", nameField), BorderLayout.NORTH);
+		new ResponsiveJText(nameField)
+				.addKeyListener((e) -> onOkayClicked())
+				.addKeyListener(KeyEventType.KEY_RELEASED, KeyEvent.VK_ENTER, (e) -> descriptionField.grabFocus());
+
 		descriptionField.setLineWrap(true);
 		descriptionField.setWrapStyleWord(true);
 		fieldsPanel.add(new LabelledField(
 				"Description",
-				new JScrollPane(descriptionField, ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED, ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER)
-		), BorderLayout.CENTER);
+				new JScrollPane(descriptionField, ScrollPaneConstants.VERTICAL_SCROLLBAR_AS_NEEDED,
+						ScrollPaneConstants.HORIZONTAL_SCROLLBAR_NEVER)),
+				BorderLayout.CENTER);
+		new ResponsiveJText(descriptionField).addKeyListener((e) -> onOkayClicked());
```

Comment:
Again, it seems a bit strange to instantly close the dialog any time a key is pressed.

Reply:
The default currently is ``ctrl + enter``, which I assume isn't generally pressed by accident, so it should be okay. I can change the default though, if you want.